src/baseWidget.js

```diff
@@ -106,7 +108,7 @@ class StylaWidget
                     offset      = 0,
                     imageSize   = 400,
                     storiesApi  = false,
-                    slug        = false,
+                    slug,
```

Comment:
Style wise I reckon moving all shorthand object members to the top makes sense, e.g.:

```js
let x = {
    foo,
    bar,
    answer : 42,
    truth  : false,
}
```

Do whatever you want though :)